Makefile

```diff
@@ -70,6 +70,14 @@ installdeps:
 	pip install --upgrade pip -q
 	pip install -e . -q
 
+SITE_PACKAGES_DIR=$$(python -c 'import site; print(site.getsitepackages()[0])')
```

Comment:
When I first tried this out in a fresh environment, I got the infamous `no such file ...prophet_model.bin` error when I tried running the unit tests. 

I was really confused about why this happened, but when I looked at the pip install, I saw:

```
Collecting prophet==1.0.1
  Using cached prophet-1.0.1-py3-none-any.whl
```

This stood out to me because as we all know by now, prophet does not publish any wheels! So I was confused why we weren't building from source.

Then I read the [pip documentation](https://pip.pypa.io/en/stable/topics/caching/#locally-built-wheels), and I learned that locally built wheels are added to the cache. 

So my guess as to what went wrong for me is that I had installed prophet beforehand without a cmdstan backend and then subsequent installs with a cmdstan backend were using the cached version so it was not properly setting up cmdstan.

So in short, I think we need to do `pip install --no-cache-dir prophet==1.0.1` instead. That properly set up the cmdstan backend and I was able to get the unit tests to pass.

 Apparently, `--no-cache-dir` is discouraged, but the [workaround in the pip docs](https://pip.pypa.io/en/stable/topics/caching/#avoiding-caching) did not work for me.